src/ui/about.js

```diff
@@ -78,18 +81,18 @@ function About(props) {
           involved in the recording.
         </Styled.p>
 
+        <Styled.h2>Support</Styled.h2>
+        <Styled.p>
+          If you are experiencing any difficulties or found any bugs,
+          please take a look at the{' '}
+          <Styled.a href="https://github.com/elan-ev/opencast-studio/issues">
+            issue tracker on GitHub
+          </Styled.a>.
+          Before filing a new issue, please check if one about your topic already exists.
+          We regularly check incoming issues and do our best to address bugs in a timely manner.
+        </Styled.p>
```

Comment:
Didn't we intentionally hide this part for all non-`studio.opencast.org` versions? I remember us being worried the issue tracker being flooded with tons of support requests from very inexperienced users? In particularly "experiencing any difficulties" could really make many people creating an issue. I wouldn't mind a bit more traffic in our issue tracker, but it would be bad if users from universities first tried creating an issue before asking their support team.

Or is that likely not a problem? I mean... I guess many inexperienced users already use `studio.opencast.org` and we weren't flooded yet... Just wanted to bring this up again.

Reply:
Valid argument. I don't know. You decide :D

Reply:
Yeah, let's hide it. The GitHub repo is linked anyway, so people who know about GitHub already know what to do.